tutorials/admin/getting-started/join-network.md

```diff
@@ -1 +1,28 @@
-# join-network
+# Joining the Network
+
+Once you have [installed](installation.md) your Marble node on your server, it is time to connect it to other Marble
+nodes. 
+
+This will allow users of your node to discover data and services available on other nodes and will allow other users in
+the Marble community to make use of your node as well. By joining the network, you contribute to the Marble community
+and help make computational climate science more accessible and collaborative. Welcome!
+
+## Add your information to the registry
+
+To join the network, you need to add your node's information to the 
+[Marble Node Registry](https://github.com/DACCS-Climate/Marble-node-registry). This is a registry of all nodes in the
+network and is used by various tools and components of each Marble node to know how to pass information between nodes.
+
+To add your node's information, make a pull request to the 
+[Marble Node Registry](https://github.com/DACCS-Climate/Marble-node-registry) repository that updates the 
+`node_registry.json` file with your node's information.
+
+To see which information should be included see the instructions 
+[here](https://github.com/DACCS-Climate/Marble-node-registry/blob/main/README.md#add-or-update-information-about-a-marble-node).
+
+```{note}
+Updating the `node_registry.json` file requires some knowledge on how to edit [JSON](https://www.json.org) files. 
+
+If you do not have the knowledge to do this, you can also provide the information in another format and one of the
```

Comment:
I think we should be more explicit about what we are asking here. I think you are suggesting that they create a new plain-text file and include information there. Did I get that right?

Reply:
My thought here is that if they expect to struggle editing a JSON file, then they probably shouldn't be the ones deploying the stack anyways.

Reply:
I'm basically saying that they can make a PR with the information in any format and we'll help them format it properly.

> My thought here is that if they expect to struggle editing a JSON file, then they probably shouldn't be the ones deploying the stack anyways.

Yeah you're probably right. Maybe I'll just remove the offer to help.